diff --git a/examples/Importers/ImportMeshUtility/b3ImportMeshUtility.cpp b/examples/Importers/ImportMeshUtility/b3ImportMeshUtility.cpp
index 442a2f7..0e28b67 100644
--- a/examples/Importers/ImportMeshUtility/b3ImportMeshUtility.cpp
+++ b/examples/Importers/ImportMeshUtility/b3ImportMeshUtility.cpp
@@ -66,7 +66,7 @@ bool b3ImportMeshUtility::loadAndRegisterMeshFromFileInternal(const std::string&
 							meshData.m_textureWidth = 0;
 							meshData.m_textureHeight = 0;
 						}
-
+                        break;
                     } else
                     {
                         b3Warning("not found [%s]\n",relativeFileName);
